File: C/oddities.c
#include <stdio.h>

void oddities(int n){
  int c;
  for(int i = 0; i < n; i++){
    scanf("%d", &c);
    if(c%2 == 0){
      printf("%d is even\n", c);
    }
    else{
      printf("%d is odd\n", c);
    }
  }
}

int main(){
  int n;
  scanf("%d", &n);

  oddities(n);
  return 0;
}

File: Javascript (NODE JS)/oddities.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

cases = 0
rl.on('line', (line) => {
    if (cases == 0){
        cases = parseInt(line)
    }
    else{
        x = parseInt(line)
        if (x%2 == 0){
            console.log(`${x} is even`)
        }
        else{
            console.log(`${x} is odd`)
        }
    }
});
